test/e2e/service.go

```diff
@@ -523,6 +525,16 @@ var _ = Describe("Services", func() {
 		tcpIngressIP := getIngressPoint(&tcpService.Status.LoadBalancer.Ingress[0])
 		Logf("TCP load balancer: %s", tcpIngressIP)
 
+		if providerIs("gce", "gke") {
+			By("demoting the static IP to ephemeral")
+			if staticIPName != "" {
+				// Deleting it after it is attached "demotes" it to an
+				// ephemeral IP, which can be auto-released.
+				deleteGCEStaticIP(staticIPName)
```

Comment:
It seems worth at least logging the error returned here (and in the defer, for that matter)